### DISCUSSION TITLE
Make slider permanent

### DISCUSSION THREAD

**Discussion by kacago22** [xournalpp/xournalpp#5831] [Ideas]
The vertical slider bar on the right side of the page only appears when the mouse/stylus gets moved over it - and it is quite slim too. Now this is very stylish, but not very user friendly.  I am having difficulty predicting where the sliding area of the slider will be when my document is long. The area that the slider covers is not used for anything else, so making it permanent would not take away anything.  Thank you!

**Comment by GithubAnon0000** [comment]
You might want to go to `preferences > View > Scrollbars` and turn on "disable scrollbar fade out". That way it won't disappear.

**Reply by kacago22** [reply]
Indeed! Thanks a lot!

**Comment by kacago22** [comment]
go to preferences > View > Scrollbars and turn on "disable scrollbar fade out". That way it won't disappear.
